misc/ERD.json

```diff
@@ -0,0 +1,64 @@
+// users
+{
+  "user_id": 1,
+  "username": "agao",
+  "password": "12345",
+  "full_name": "Andy Gao",
+  "email": "agao@andrew.cmu.edu",
+  "address": "5023 Forbes Ave",
+  "phone_number": 9011234567,
+  "birthday": 12041997,
+  "favorite_donut": "pineapple",
+  "credit_card": 1234567898765432
+}
+// customers
+{
+  "customer_id": 1,
+  "user_id": 1,
+  "address": "5023 Forbes Ave",
+  "birthday": 12041997,
+  "fav_donut": "pineapple",
+  "credit_card": 1234567898765432
+}
+// employees
+{
+  "employee_id": 1,
+  "user_id": 1,
+  "position": "manager"
```

Comment:
I wonder if we should have a "permissions" field. For example, employees might need different combinations of being able to fulfill orders, cancel orders, perform drone maintenance, etc. I'm worried that employees could also have very different titles ("assistant manager", "shift manager", "primary cashier", etc.), which will make it difficult to case on position title.